src/apps/engine/src/main.cpp

```diff
@@ -95,7 +96,8 @@ void HandleWindowEvent(const storm::OSWindow::Event &event)
         if (core_private->initialized())
         {
             core_private->AppState(bActive);
-            if (const auto soundService = static_cast<VSoundService *>(core.GetService("SoundService")))
+            if (const auto soundService = static_cast<VSoundService *>(core.GetService("SoundService"));
```

Comment:
oof.. we need to move this into sound service... but not as part of this PR of course

Reply:
Yes, very much agree